xbmc/addons/binary-addons/BinaryAddonManager.cpp

```diff
@@ -55,6 +57,20 @@ bool CBinaryAddonManager::Init()
   for (auto addon : binaryAddonList)
     AddAddonBaseEntry(addon);
 
+  CServiceBroker::GetAddonMgr().RegisterAddonMgrCallback(ADDON_ADSPDLL, this);
```

Comment:
Well this is because @AlwinEsch tries to implement a new manager for binary add-ons as it was not wanted to merge them.